### PR TITLE
Newfolder hotfix

### PR DESCRIPTION
**Pull request by ipendlet**

### CODE REVIEW

File: capture/devconfig.py [hunk 1]
```diff
@@ -116,4 +116,4 @@
 
 # must be 'default' or 'wolfram'
 # 'wolfram' is currently experimental and unsupported
-sampler = 'default'
+sampler = 'wolfram'
```
Comment (aadharna) [inline capture/devconfig.py]: Shouldn't default stay default?

Comment (ipendlet) [reply capture/devconfig.py]: We pull wolfram in via requirements.  Everyone who is currently using is expecting wolfram... We might want to encode  fallback, but this was intentional.  I am actually MORE worried that they WON"T use wolfram.

Comment (ipendlet) [reply capture/devconfig.py]: Like, if we run experiments using teh default sampler, that will be problematic.  We must hope that the center holds for wolfram.

Comment (aadharna) [reply capture/devconfig.py]: Not worried about MIT?

Comment (aadharna) [reply capture/devconfig.py]: Installing the wolframclient library in the requirements.txt does *not* install the wolfram kernel, which it needs to run.

Comment (ipendlet) [reply capture/devconfig.py]: Let's leave default and I will just remember to tell the people what we are using.  I will also work on a code fix to write out which sampler is being used to the final observation interface (model and author columns)

**Pull request merged**